### PR TITLE
Handle multiple URIs like KeePassDX

### PR DESCRIPTION
**Pull request by diogotito**
This handles entries with more than one URI in a similar way to [KeePassDX](https://github.com/Kunzisoft/KeePassDX/), one of the two popular Android applications for KeePass databases.

It works like this for website URLs:

- The first URL gets stored in the standard `URL` property
- The second goes to a `URL_1` custom property
- The third goes to `URL_2`, and so on...

URIs starting with `androidapp://` and `iosapp://` get treated specially, as those are what Bitwarden's mobile app uses for its auto-fill mechanism on Android and Apple devices, respectively.
I mapped Android URIs to `AndroidApp`, `AndroidApp_1`, etc. as that's what KeePassDX uses for auto-fill.
I don't know what to do with iOS URIs though, so I just stored them in `iOS app #N` custom properties.

### CODE REVIEW

File: bitwarden-to-keepass.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def bitwarden_to_keepass(args):
                 except Exception as e:
                     if 'already exists' in str(e):
                         is_duplicate_title = True
-                        continue
+                        break
```
Comment (davidnemec) [inline bitwarden-to-keepass.py:67]: Weird that you ended up in infinite loop. It should not happen - well, at least not due to having multiple items with the same name. `continue` here saves the same item with bitwardens uuid appended to the name. 
https://github.com/davidnemec/bitwarden-to-keepass/blob/9c27ba943c52558622b6108bb84377dbce46adb8/bitwarden-to-keepass.py#L54

Can you maybe check what is causing this?

Anyways, using `break` here will result in missing items in KeePass export.

Comment (diogotito) [reply bitwarden-to-keepass.py:67]: You're right, `break` does indeed skip some items in the KeePass export, I only noticed that now.
I'll try to find out what is causing `continue` to come back to the same `'already exists'` error. The script ran fine before I touched it so it must be something on my end.

Comment (diogotito) [reply bitwarden-to-keepass.py]: I didn't manage to replicate the infinite loop. My experiments only confirmed that this logic does indeed work as intended.
I decided to revert this commit.

**Comment by diogotito** [comment]
Oops, sorry for the mess. I've just reset my `multiple_urls` branch to your `master` and produced a single commit this time.
I took the chance to extract my code to a separate method with the intention to make it more consistent with the rest of the script. I think it's all good now.

**Comment by davidnemec** [comment]
Thanks for your time!

**Pull request merged**